turbo.json

```diff
@@ -1,22 +1,14 @@
 {
   "$schema": "https://turborepo.org/schema.json",
```

Comment:
suggestion: should we update this to: `https://turbo.build/schema.json`. Don't think it has any practical implications, but for hygiene :)

Reply:
done!